File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_python/p02577/p02577_num5_parsed.py
# Program to determine whether a number N is a multiple of 9

# Function to find the sum of digits of a number
def get_sum_of_digits(n):
    sum = 0
    while n > 0:
        sum += n % 10
        n //= 10
    return sum

# Read the input N
N = int(input())

# Find the sum of digits of N
sum_of_digits = get_sum_of_digits(N)

# Check if the sum is a multiple of 9
if sum_of_digits % 9 == 0:
    print("Yes")
else:
    print("No")

File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_cpp/p02577/p02577_num5_parsed.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
    string N;
    cin >> N;

    int sum = 0;
    for (char c : N) {
        sum += c - '0';
    }

    if (sum % 9 == 0) {
        cout << "Yes" << endl;
    } else {
        cout << "No" << endl;
    }

    return 0;
}